go/autogen/api/models.go

```diff
@@ -117,6 +123,7 @@ type OllamaClientConfiguration struct {
 	ModelCapabilities interface{}       `json:"model_capabilities,omitempty"`
 	ModelInfo         *ModelInfo        `json:"model_info"`
 	Options           map[string]string `json:"options"`
+	BaseClientConfig
```

Comment:
This won't work for `Ollama`: see: https://github.com/microsoft/autogen/blob/f0b73441b6dde66616ce4400f11e994eaa7403f7/python/packages/autogen-ext/src/autogen_ext/models/ollama/config/__init__.py#L40